index.bs

```diff
@@ -637,6 +637,7 @@ The <dfn method for=CookieStore>get(|options|)</dfn> method steps are:
 1. Let |origin| be the [=current settings object=]'s [=/origin=].
 1. If |origin| is an [=opaque origin=], then return [=a promise rejected with=] a "{{SecurityError}}" {{DOMException}}.
 1. Let |url| be the [=current settings object=]'s [=creation URL=].
+1. If |options| is empty, then return [=a promise rejected with=] a "{{TypeError}}".
```

Comment:
No quotes here for `TypeError`

The quotes are used for "`SecurityError`" because it's a string... the actual exception type is `DOMException`.